canonical/jp.ubuntu.com
# WD-11646 Refresh ai-ml page

## Done

 Refreshed ai-ml page
## QA

-  Please run the site locally to see the updates.
- [Copydoc](https://docs.google.com/document/d/186nJQj0P5NTZR3CKnPkjq5I8kupCy1boHnOoHQZAHNI/edit)
- [Figma](https://www.figma.com/design/tI8YK4wQnKiWLAxdX1fWnB/24.10-jp.ubuntu.com?node-id=313-7311&t=JGKSeCRmztoC8QA9-0)
- [Demo](https://jp-ubuntu-com-490.demos.haus/ai-ml)
## Issue / Card

Fixes # [WD-11646](https://warthogs.atlassian.net/browse/WD-11646)


[WD-11646]: https://warthogs.atlassian.net/browse/WD-11646?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
